.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: "Spell check"
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  check:
+    # run on all push events or on PR syncs not from the same repo
+    if: github.event_name != 'pull_request' || github.repository != github.event.pull_request.head.repo.full_name
```

Comment:
Does this need to be GitHub.repository.full_name to check the name string, not the object itself?

Reply:
`github.repository` is already a string and it's used like that in the Paper repo, so should be fine

Reply:
Ah ok, it's not like the webhook event obj